src/components/Card/index.tsx

```diff
@@ -29,30 +30,41 @@ type CardProps = {
 function Card({ cardData }: CardProps) {
   return (
     <div className={styles.cardBox}>
-      {cardData.imageUrl && (
+      {cardData?.imageUrl && (
         <Image
-          src={cardData.imageUrl}
+          className={styles.cardImage}
+          src={cardData?.imageUrl}
           alt="카드 배경 이미지"
           width={274}
           height={160}
         />
       )}
-      <span>{cardData.title}</span>
-      <ul className={styles.tags}>
-        {cardData.tags?.map((tag, index) => <li key={index}>{tag}</li>)}
-      </ul>
-      <div className={styles.bottom}>
-        <div className={styles.date}>
-          <Image
-            src="/svgs/calendar.svg"
-            alt="달력 아이콘"
-            width={18}
-            height={18}
-          />
-          <p>{cardData.dueDate}</p>
+      <div className={styles.desktopBox}>
+        <div className={styles.tabletBox}>
+          <span>{cardData?.title}</span>
+          <div className={styles.tabletContent}>
+            <ul className={styles.tags}>
+              {cardData?.tags?.map((tag, index) => (
+                <li key={index}>
+                  <Tag>{tag}</Tag>
+                </li>
+              ))}
+            </ul>
+            <div className={styles.bottom}>
+              <div className={styles.date}>
+                <Image
+                  src="/svgs/calendar.svg"
+                  alt="달력 아이콘"
+                  width={18}
+                  height={18}
+                />
+                <p>{cardData?.dueDate?.split(' ')[0]}</p>
+              </div>
+            </div>
+          </div>
         </div>
-        <div>
-          <ProfileIcon profile={cardData.assignee || null} small />
+        <div className={styles.profileIcon}>
+          <ProfileIcon profile={cardData?.assignee || null} small />
```

Comment:
```suggestion
          <ProfileIcon profile={cardData?.assignee || null} small />
```
여기에 null이 들어갈 수 있나요??